lib/external-secret.js

```diff
@@ -26,18 +26,23 @@ async function startWatcher ({
   kubeClient,
   customResourceManifest,
   logger,
-  eventQueue
+  eventQueue,
+  scopeNamespace = null
```

Comment:
Do we need the `null` default here?

Reply:
We don't necessarily need it but I think its cleaner to show that its optional and might be null instead of having it end up being `undefined`